Makefile

```diff
@@ -56,7 +57,9 @@ clean-docs:
 
 test:
 	$(PYTHON) $(TESTRUNNER)
+	$(PYTHON) $(PLUGIN_TESTRUNNER) --device=default.qubit.autograd
 
 coverage:
 	@echo "Generating coverage report..."
 	$(PYTHON) $(TESTRUNNER) $(COVERAGE)
+	$(PYTHON) $(PLUGIN_TESTRUNNER) --device=default.qubit.autograd $(COVERAGE) --cov-append
```

Comment:
❤️ 
We should begin deleting all the repeated device integration tests, and just add lines to the makefile here